elm/tests/Test/Reporter/Console/FormatTest.elm

```diff
@@ -58,4 +60,58 @@ suite =
                     highlightEqual expected actual
                         |> Expect.notEqual Nothing
             ]
+        , Test.fuzzWith
+            { runs = 10000
+            , coverage =
+                Test.reportCoverage
+                    [ ( "fizz", \n -> (n |> modBy 3) == 0 )
+                    , ( "buzz", \n -> (n |> modBy 5) == 0 )
+                    , ( "even", \n -> (n |> modBy 2) == 0 )
+                    , ( "odd", \n -> (n |> modBy 2) == 1 )
+                    ]
+            }
+            (Fuzz.intRange 1 20)
+            "Fizz buzz even odd - fail"
+            (\n -> Expect.fail "boo")
```

Comment:
elm-review is flagging `n` as unused in this and the following tests.

- Should it be used in an expectation?
- Are `"boo"` and `"x"` the best strings to fail with?

Reply:
I'll make the strings a bit more meaningful :+1: and change `\n` into `\_`